RealismOverhaul.netkan

```diff
@@ -23,30 +23,29 @@
         { "name" : "B9-PWings-Fork" },
         { "name" : "ConnectedLivingSpace" },
         { "name" : "DeadlyReentry" },
-        { "name" : "HangarExtender" },
+        { "name" : "HangerExtenderExtended" },
         { "name" : "KSP-AVC" },
         { "name" : "MechJeb2" },
         { "name" : "ProceduralFairings" },
         { "name" : "ProceduralFairings-ForEverything" },
```

Comment:
Last release was for 1.2.2 and maintained by RO team

Reply:
will look into updating or removing

Reply:
I have no permissions for this and the people who do are awol so before release I may remove this.